jina/drivers/reduce.py

```diff
@@ -29,13 +29,22 @@ def __call__(self, *args, **kwargs):
         self._traverse_apply(self.docs, *args, **kwargs)
         self.doc_pointers.clear()
 
+    def _apply_root(self, docs: 'DocumentSet', field: str, *args, **kwargs):
+        docs = []
+        for doc in self.docs:
+            docs.append(doc)
+        request = self.msg.request
+        request.body.ClearField(field)
+        request.docs.extend(docs)
```

Comment:
so I guess you saw the problem I had?

Reply:
@JoanFM The problem was that the docs were stored in the `body.docs` as well as in `docs`?